dappnode_package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "nimbus-gnosis.dnp.dappnode.eth",
   "version": "0.1.0",
-  "upstreamVersion": "v23.5.1",
+  "upstreamVersion": "v23.11.0",
```

Comment:
Please update to [v24.1.1](https://github.com/orgs/gnosischain/packages/container/gnosis-nimbus-eth2/164812888?tag=v24.1.1)